apps/api/src/routes/applicationRouter.ts

```diff
@@ -287,4 +285,51 @@ applicationRouter.post('/reject', jsonParser, async (req, res) => {
 	}
 });
 
+// POST: Submit revisions
+applicationRouter.post('/:applicationId/submit-revision', jsonParser, async (req, res) => {
```

Comment:
This should use `withParamsSchemaValidation`

Reply:
See example in `collaboratorsRouter` / List Collaborators
Schema can copy `collaboratorsListParamsSchema`